src/ai/backend/client/func/quota_scope.py

```diff
@@ -0,0 +1,156 @@
+import textwrap
+from typing import Any, Sequence
+
+from ..output.fields import group_fields, quota_scope_fields, user_fields
+from ..output.types import FieldSpec
+from ..session import api_session
+from ..types import set_if_set
+from .base import BaseFunction, api_function
+
+_default_user_fields = (
+    user_fields["uuid"],
+    user_fields["username"],
+)
+
+_default_project_fields = (
+    group_fields["id"],
+    group_fields["name"],
+)
+
+_default_detail_fields = (
+    quota_scope_fields["usage_bytes"],
+    quota_scope_fields["usage_count"],
+    quota_scope_fields["hard_limit_bytes"],
+)
+
+_default_quota_scope_fields = (
+    quota_scope_fields["quota_scope_id"],
+    quota_scope_fields["storage_host_name"],
+)
+
+
+class QuotaScope(BaseFunction):
+    @api_function
+    @classmethod
+    async def get_user_qsid(
```

Comment:
I think of 3 options
1. Rename this function to `get_user_info`
2. Use a function in `client.func.user.User`. There will be a minor update to the function in `client.func.user.User`.
3. Return full quota scope id like `user:fda...` rather than returning only `id`. Because the name of this function is `get_qsid` not just `get_id`.

Reply:
> I think of 3 options
> 
> 1. Rename this function to `get_user_info`
> 2. Use a function in `client.func.user.User`. There will be a minor update to the function in `client.func.user.User`.
> 3. Return full quota scope id like `user:fda...` rather than returning only `id`. Because the name of this function is `get_qsid` not just `get_id`.

Thanks. I would change the name of this function to `get_user_info`.